packages/vc-test-suite/__tests__/httpService.test.ts

```diff
@@ -0,0 +1,32 @@
+import { request } from '../httpService';
+import axios from 'axios';
+
+jest.mock('axios');
+
+const mockedAxios = axios as jest.Mocked<typeof axios>;
+
+describe('httpService', () => {
```

Comment:
Should we add an additional test case for when the `request` function is executed without the `headers` in the `params`?